trnsysGUI/pythonInterface/regimeExporter/renderDiagramOnPDFfromPython.py

```diff
@@ -44,20 +48,22 @@ def temperaturesPrintFilePath(self) -> _pl.Path:
 
     def export(
         self, onlyTheseRegimes: _tp.Optional[_tp.Sequence[str]] = None
-    ) -> None:
+    ) -> _tp.List[str]:
 
         if not onlyTheseRegimes:
-            self._makeDiagramFiles()
+            self._makeNoFlowDiagramFiles()
 
         regimeValues = _gdr.getRegimes(
             self.projectDir / self.regimesFileName, onlyTheseRegimes
         )
         relevantNames = list(regimeValues.columns)
         relevantBlockItems = self.getRelevantBlockItems(relevantNames)
 
-        self._simulateAndVisualizeMassFlows(relevantBlockItems, regimeValues)
+        return self._simulateAndVisualizeMassFlows(
+            relevantBlockItems, regimeValues
+        )
 
-    def _makeDiagramFiles(self, regimeName: str = "diagram") -> None:
+    def _makeNoFlowDiagramFiles(self, regimeName: str = "diagram") -> None:
```

Comment:
def _makeNoFlowDiagramFiles ...
   """Wrapper function to make intention clearer."""
   failures = _makeDiagramFiles...
   return failures

Reply:
At some point, we might have a conflict here, which would need the flows to be removed first to get this properly.
Not a problem to solve now.